test/WebSites/BasicWebSite/Controllers/HomeController.cs

```diff
@@ -16,7 +16,7 @@ public IActionResult Index()
             return View();
         }
 
-        public IActionResult PlainView()
+        public object PlainView()
```

Comment:
Could you call this out in the test (saying it's specifically done this way to ensure 4960 works)?